PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/configuration/ui/AzureFunctionsConfigurationPanel.kt

```diff
@@ -173,9 +174,13 @@ class AzureFunctionsConfigurationPanel: AzureRiderAbstractConfigurablePanel {
 
     override fun doOKAction() {
         if (coreToolsPathField.text != "" && File(coreToolsPathField.text).exists()) {
+
+            val coreToolsPath = FunctionsCoreToolsInfoProvider
```

Comment:
Hm. Sorry, that's not exactly what I meant :) I thought about validation only (so, you get some error that path does not exist). From UX side it might be strange if we change user's the value. I would rather revert this back and add some info in logs about path that is used for running core tools. Thoughts?

Reply:
How about the validation updates the value and shows a message about the fact that we updated it?

Reply:
Reverted, thanks!